uber/custom_tags.py

```diff
@@ -543,7 +543,7 @@ def _notice(self, label, takedown, amount_extra, discount):
         else:
             for day, price in sorted(c.PRICE_BUMPS.items()):
                 if day < takedown and localized_now() < day:
-                    return '<div class="prereg-price-notice">Price goes up to ${} at 11:59pm {} on {}</div>'.format(price - discount + int(amount_extra), (day - timedelta(days=1)).strftime('%Z'), (day - timedelta(days=1)).strftime('%A, %b %e'))
+                    return '<div class="prereg-price-notice">Price goes up to ${} at 11:59pm {} on {}</div>'.format(price - int(discount) + int(amount_extra), (day - timedelta(days=1)).strftime('%Z'), (day - timedelta(days=1)).strftime('%A, %b %e'))
```

Comment:
FWIW I did look and the generate INI file on the prod server does look like all the other ints in ubersystem, and the configspec.ini looks right too:

configspec.ini
```
# Group badge prices always equal the attendee badge prices minus this discount.
group_discount = integer(default=10)
```

```
group_discount = 5
```